lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -129,6 +130,43 @@ def configure(conf)
       log_level = conf['@log_level'] || conf['log_level']
       log.warn "Consider to specify log_level with @log_level." unless log_level
     end
+
+    if @dlq_handler && @dlq_handler.is_a?(Hash)
+      dlq_type = @dlq_handler[:type]||@dlq_handler['type']
+      if dlq_type
```

Comment:
Could you use early return style for readability?

Reply:
I will extract to its own configure method.  Early return doesn't make sense in the current impl given future changes would require configuration to occur before this block; early return could affect how someone else adds